vehicle/constants.py

```diff
@@ -0,0 +1,30 @@
+import enum
+
+FRONT_CAMERA_INDEX = 0
+BOTTOM_CAMERA_INDEX = 1
+BACK_CAMERA_INDEX = 2
+
+BACKWARD_CAM_INDICES = (2,)
```

Comment:
Do you think it might be better to be able to specify these in the camera configuration file so that we don't have to have a specific ordering?